lib/files.cf

```diff
@@ -426,7 +426,7 @@ bundle edit_line manage_variable_values_ini(tab, sectionName)
 
       # match a line starting like the key something
       "$(index)\s*=.*"
-      edit_field => col("=","2","$($(tab)[$(sectionName)][$(index)])","set"),
+      edit_field => col("\s*=\s*","2","$($(tab)[$(sectionName)][$(index)])","set"),
```

Comment:
@mikeweilgart This does not seem to have an effect on the test.

Reply:
@mikeweilgart I believe that these changes to `manage_variable_values_ini` have no effect since this bundle manages the complete contents of the section.

This [delete_lines promise](https://github.com/cfengine/masterfiles/pull/1609/files#diff-aaead9adbc8b4f10c3e8dc460612d0ccR434-R437) will empty the section first, then `insert_lines` will populate the entire section and there will never be any opportunity for the `field_edits` to make any adjustments.

You said you tested in your environment, did you `test manage_variable_values_ini`, or `just set_variable_values_ini`?

I am inclined to redact the changes to `manage_variable_values_ini` since they have no effect.

Reply:
I only tested `set_variable_values_ini`.  However, the *only* difference between the two bundles is the `delete_lines` promise in one of them.  I think it would be strange to let them diverge in other respects.

Reply:
thanks for clarifying that you tested set_variable_values_ini. I am redacting the changes to manage_variable_values_ini and I will squash this down into 2 commits. One extending the test to show the behavior before the change, and then your changes along with updates to the test.